include/nbl/video/IPhysicalDevice.h

```diff
@@ -126,17 +225,98 @@ class IPhysicalDevice : public core::Interface, public core::Unmovable
                 return static_cast<uint32_t>(core::min<uint64_t>(infinitelyWideDeviceWGCount,maxResidentWorkgroups));
             }
         };
-        const SLimits& getLimits() const { return m_limits; }
+        
+        struct SProperties
+        {
+            //--> VkPhysicalDeviceProperties:
+            APIVersion  apiVersion;
+            // uint32_t driverVersion;
+            // uint32_t vendorID;
+            // uint32_t deviceID;
+            E_TYPE      deviceType;
+            // char     deviceName[VK_MAX_PHYSICAL_DEVICE_NAME_SIZE];
+            // uint8_t  pipelineCacheUUID[VK_UUID_SIZE];
```

Comment:
expose, just fill `driverVersion,vendorID,deviceID` with bogus values on GL (all zeros?)

on OpenGL I guess `deviceName=getGetString(GL_RENDERER)` and `pipelineCacheUUID` could be a hash (we hasve some 256bit hash function in `core`) of that string